llvm/docs/GitHub.rst

```diff
@@ -15,6 +15,12 @@ The LLVM Project uses `GitHub <https://github.com/>`_ for
 This page describes how the LLVM Project users and developers can
 participate in the project using GitHub.
 
+Before your first PR
+====================
+
+Please ensure that you have set a valid email address in your GitHub account,
+see :ref:`github-email-address`.
```

Comment:
Also kinda weird that we link out of a page dealing with GitHub to talk about GitHub, but it is part of https://llvm.org/docs/DeveloperPolicy.html#github-email-address which deals with policies and the one you're editing is about submitting code.

So it's fine to link here and leave the content where it is.

Reply:
Agreed, I see the "email address" policy as something higher-level than "submitting patches via GitHub" 👍🏻